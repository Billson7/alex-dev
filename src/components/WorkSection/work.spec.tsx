import React from "react"
import { render } from "@testing-library/react"
import { Work } from "."

test("renders Work section correctly", () => {
  const { container } = render(<Work heading='heading' caption="caption" intro='intro' />)

  expect(container).toMatchSnapshot()
})
