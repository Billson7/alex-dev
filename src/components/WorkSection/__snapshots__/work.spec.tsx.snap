// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Work section correctly 1`] = `
<div>
  <section
    class="section section-work"
  >
    <div
      class="flex-row"
    >
      <h2
        class="profile-heading"
      >
        heading
      </h2>
      <p
        class="profile-caption-invert"
      >
        caption
      </p>
      <p
        class="profile-intro"
      >
        intro
      </p>
      <code
        style="display: none;"
      >
        github things? screenshots? links?
      </code>
    </div>
  </section>
</div>
`;
